Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -803,7 +803,8 @@ public virtual eImageType ItemImageType
         {
             get
             {
-                throw new NotImplementedException("ItemImageType not defined for: " + this.GetType().FullName);                
+                //throw new NotImplementedException("ItemImageType not defined for: " + this.GetType().FullName); 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5576541)